### ISSUE TITLE
Broken formula parsing in v0.6+

### ISSUE THREAD

**Issue by dakota** [NeonXP/MathExecutor#43]
I have a formula that works perfectly fine in v0.5, but in v0.6 and v0.7 it breaks in different ways. Formula is: `round(100 * 1.10, 2) - 0.01`, expected result is `109.99`. v0.6+ gives a result of `99.99`.

If I change the formula to `round((100 * 1.10), 2) - 0.01`, then v0.6+ gives the correct value.

The code is:

```php
        $calculator = new MathExecutor();

        $calculator->addFunction('round', function ($decimals, $value) {
            return round($value, $decimals);
        }, 2);

        $result = $calculator->execute('round(100 * 1.10, 2)');
```

**Comment by phpfui** [comment]
Walther, we fixed your issue. Turns out we are not respecting the comma operator correctly.  Your formulas now execute the same as PHP would execute them.

Also note that in 0.7.1 we fixed the ordering of function parameters.  They were backwards.  I noticed your example had them backward (which was correct before 0.71, but now is incorrect).  This only matters if you depend on ordering, which of course round does.  But min and max don't care obviously.

Alex should release this tomorrow morning his time.  If not, I will do it before noon New York time.

Thanks for the heads up and let us know if you find any other issues.
